tests/Tests/400__ods_dashboard/411__ods_dashboard_settings_pvc_size.robot

```diff
@@ -107,8 +108,6 @@ Change And Apply PVC size
     ...    and restart the Notebook controller POD
     [Arguments]     ${size}
     Check If PVC Change Is Permanent    ${size}
-    Run Keyword And Ignore Error    Delete Pods Using Label Selector    redhat-ods-applications    app=notebook-controller
-    Wait Until Keyword Succeeds    15 times    1 sec   Verify Operator Pod Status    redhat-ods-applications    app=notebook-controller
```

Comment:
why are these useless?

Reply:
We don need those ..once we make changes to ODHdahsboard config the configuration is automatically load after sometime.